src/get-selectors-and-clean.sh

```diff
@@ -4,14 +4,19 @@ OUTPUT_DIR=$1
 
 SELECTOR=""
 if ! git log -m -1 --name-only --pretty=format: | grep -qv "^helmfiles/"; then
-  for namespace in $(git log -m -1 --name-only --pretty=format: | grep "^helmfiles/" | cut -d "/" -f 2 | sort -u); do
-    SELECTOR="${SELECTOR} --selector namespace=${namespace}"
-    rm -rf ${OUTPUT_DIR}/cluster/namespaces/${namespace}.yaml
-    rm -rf ${OUTPUT_DIR}/cluster/resources/${namespace}
-    rm -rf ${OUTPUT_DIR}/customresourcedefinitions/${namespace}
-    rm -rf ${OUTPUT_DIR}/namespaces/${namespace}
-  done
-  >&2 echo helmfile with selector ${SELECTOR}
+  changedNamespaces=$(git log -m -1 --name-only --pretty=format: | grep "^helmfiles/" | cut -d "/" -f 2 | sort -u)
+    if echo "$changedNamespaces" | grep -q ^jx$; then
+        >&2 echo jx namespace changed, no selectors added
+    else
+      for namespace in ${changedNamespaces}; do
+        SELECTOR="${SELECTOR} --selector namespace=${namespace}"
+        rm -rf ${OUTPUT_DIR}/cluster/namespaces/${namespace}.yaml
+        rm -rf ${OUTPUT_DIR}/cluster/resources/${namespace}
+        rm -rf ${OUTPUT_DIR}/customresourcedefinitions/${namespace}
+        rm -rf ${OUTPUT_DIR}/namespaces/${namespace}
+      done
+      >&2 echo helmfile with selector ${SELECTOR}
+    fi
```

Comment:
Make indentation consistent. Since the file in general has an indentation of 2 spaces you should follow that.